packages/lib-core/src/app/core/reducers/index.ts

```diff
@@ -0,0 +1,11 @@
+import type { ActionType as Action } from 'typesafe-actions';
```

Comment:
Can we put any Redux related files in `redux` directory, e.g. `redux/reducers/..` ?